hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -322,16 +324,25 @@ private boolean isECBucket(BucketInfo bucketInfo) {
    * @param omVolumeArgs
    */
   private void addDefaultAcls(OmBucketInfo omBucketInfo,
-      OmVolumeArgs omVolumeArgs) {
-    // Add default acls for bucket creator.
+      OmVolumeArgs omVolumeArgs, OzoneManager ozoneManager) throws OMException {
     List<OzoneAcl> acls = new ArrayList<>();
+    // Add default acls
+    acls.addAll(getDefaultAclList(createUGIForApi(), ozoneManager.getConfiguration()));
     if (omBucketInfo.getAcls() != null) {
+      // Add acls for bucket creator.
       acls.addAll(omBucketInfo.getAcls());
     }
 
+    // Link bucket default acl
+    if (omBucketInfo.getSourceBucket() != null && omBucketInfo.getSourceVolume() != null) {
+      acls.add(LINK_BUCKET_DEFAULT_ACL);
```

Comment:
RpcClient also added, and same added here also, plz check if de-duplicated this from this point.

Reply:
make sense.